gradle/scripts/lib/java-rpc-proto.gradle

```diff
@@ -60,6 +60,12 @@ configure(projectsWithFlags('java')) {
                         }
                     }
 
+                    if (project.ext.hasFlag('krotodc-grpc')) {
```

Comment:
The original source of this code is https://github.com/line/gradle-scripts.
Could you send a PR for this change to https://github.com/line/gradle-scripts as well?

Reply:
I've sent the [PR](https://github.com/mscheong01/gradle-scripts/pull/1) containing the identical changes 🙂

Reply:
It would be nicer to mention the `krotodc-grpc` flag in the README.md 😉 
https://github.com/line/gradle-scripts#built-in-flags